File: apps/api/src/api/schemas/suggestions.py
from pydantic import BaseModel


class BulletRewrite(BaseModel):
    original: str
    suggested: str | None
    error: str | None = None


class SuggestionsResponse(BaseModel):
    ai_available: bool
    message: str | None
    bullet_rewrites: list[BulletRewrite]
    score_explanation: str | None

File: apps/web/src/types/suggestions.ts
export interface BulletRewrite {
  original: string;
  suggested: string | null;
  error?: string | null;
}

export interface SuggestionsResult {
  ai_available: boolean;
  message: string | null;
  bullet_rewrites: BulletRewrite[];
  score_explanation: string | null;
}